README.md

```diff
@@ -125,6 +125,8 @@ Most styling is included in the basic UI, but we also ship a few css files that
 1. Both the HTML and the PDF side have css that is necessary to be included for the dependencies we use to render correctly. This is built automatically into `@nypl/web-reader/dist/esm/index.css` and `@nypl/web-reader/dist/cjs/index.css`. Depending which package you are using, you should include one of those files in your bundle.
 1. The HTML Reader can inject `<style>` tags (and other tags) into the reader iframe itself, called an "injectable". This is used to add styles to the html content of the publication. More on this is below.
 
+In order for the OpenDyslexic font to be displyed in the Settings panel as intended, you must include the `fonts/opendyslexic` folder and its contents in your entry directory (in a Next.js app, this would be in your `/public` folder). You can find this folder in `@nypl/web-reader/static`.
```

Comment:
I think this is true of all the fonts (except publisher) actually, not just open dyslexic.

Reply:
I think I'm a little confused about the fonts. I'll put some time on for us to discuss this IRL!

Reply:
Expanded this comment per our conversation.